Source/Core/Core/IOS/WFS/WFSI.cpp

```diff
@@ -308,6 +309,36 @@ IPCCommandResult WFSI::IOCtl(const IOCtlRequest& request)
     WARN_LOG(IOS_WFS, "IOCTL_WFSI_DELETE_TITLE: unimplemented");
     break;
 
+  case IOCTL_WFSI_CHANGE_TITLE:
+  {
+    u64 title_id = Memory::Read_U64(request.buffer_in);
+    u16 group_id = Memory::Read_U16(request.buffer_in + 0x1C);
+
+    // TODO: Handle permissions
+    SetCurrentTitleIdAndGroupId(title_id, group_id);
+
+    // Change home directory
+    return_error_code = -3;
```

Comment:
Whats `-3`? And should that comment be further down instead of next to the return code? Or is that the failure code in case the path is too long?

Reply:
Done. I've seen many occurrences of `-3` in functions calling WFSISrv's IOCtl (most of them are stubbed by Dolphin) and indeed, it looks like a silly mistake.